### ISSUE TITLE
Develoer Guide

### ISSUE THREAD

**Issue by amitsharma8891** [sapplica/sentrifugo#123]
Hi Team,

First of all very good product, I like the features you guys are built. I wanted to do some customization and I don't have any experience about Zend framework so finding very difficult to do a small change.
So can you guys provide any developer guide so that I can do some customization? Any suggestion or help will be highly appreciated 
Thanks

**Comment by 4poorv** [comment]
Hi @sapplica,
I would like to contribute.
Can you share development instructions?

Thanks

**Comment by ch-gz** [comment]
@amitsharma8891 They are using modified Zend framework 1.11.11. If you check their source code and config.ini then you can find it's not standard ZF1 application structure.

**Comment by 4poorv** [comment]
Yes, it's 1.11.11. I confirmed with them on support portal and asked about developer guide too. That's what they said:

> In Sentrifugo, we are using Zend framework version 1.11.11 with 'Module' structure.
We are sorry to disappoint you. Currently, we don't have a developer guide/manual to provide. Thank you!

**Comment by ch-gz** [comment]
@4poorv Now I'm working on it to do some modifications.
